### ISSUE TITLE
[DF] `Describe` output does not display correctly in Jupyter notebooks

### ISSUE THREAD

**Issue by eguiraud** [root-project/root#8893]
![image](https://user-images.githubusercontent.com/10999034/130625213-022e17e3-564d-40c4-a1a2-e54fd691ab2e.png)

**Comment by ikabadzhov** [comment]
Problem here is that `Describe` defined in `tree/dataframe/inc/ROOT/RDF/RInterface.hxx` returns a string. The simplest solution is to call `print(df.Describe())` to format properly the string.

"Fixing" the desired output would change the existing usage of `Describe()` - returned type change from string to void and printing inside of the Describe.

This is what is going on:
```python
def foo(): # returns string
    return "a\nb"

def bar(): # void
    print("a\nb")

foo()                # this is our case, giving excplicitly "a\nb"
bar()                # this is good
print(foo())       # this is good
```

**Comment by eguiraud** [comment]
As per the discussion at https://mattermost.web.cern.ch/root/pl/9yqt4pm4qtfs38n3pc5stkhp5a , we want to change Describe's behavior so that by default it prints to stdout. The signature should be:

```cpp
void Describe(std::ostream &out = std::cout);
```

 We need to do this before the feature is published in v6.26.

**Comment by eguiraud** [comment]
Reopening until this is backported to v6.26
